src/funcs/transformToDbml.js

```diff
@@ -0,0 +1,38 @@
+const { EOL } = require('os');
+
+// TODO: possible to use https://www.dbml.org/js-module/#api for the transform to dbml?
```

Comment:
They use sql statements such as `create table tableName fields, fields, fields, etc.`, and not json.